hw3/index35.js

```diff
@@ -0,0 +1,50 @@
+const users = [
+    {
+      id: 1,
+      first_name: 'Jeanette',
+      last_name: 'Penddreth',
+      email: 'jpenddreth0@census.gov',
+      gender: 'Female',
+      ipAddress: '26.58.193.2',
+    },
+    {
+      id: 2,
+      first_name: 'Petr',
+      last_name: 'Jackson',
+      email: 'gfrediani1@senate.gov',
+      gender: 'Male',
+      ipAddress: '229.179.4.212',
+    },
+];
+
+
+
+const sortUsers = (users) => {
+const resultAll = {};
+resultAll.men = [];
+resultAll.women = [];
```

Comment:
why ? you can create men and women field at 23 line. no matter to do it in this way;
`const result = {men: [], women: []}`